addOns/graphql/src/test/java/org/zaproxy/addon/graphql/GraphQlFingerprinterUnitTest.java

```diff
@@ -307,13 +309,20 @@ private static String errorResponse(String error, String field, boolean data) {
                 + " }";
     }
 
+    @SuppressWarnings("null")
     @ParameterizedTest
     @MethodSource("fingerprintData")
     void shouldFingerprintValidData(String graphqlImpl, String response) throws Exception {
         // Given
         ExtensionAlert extensionAlert = mockExtensionAlert();
         nano.addHandler(new GraphQlResponseHandler(response));
         var fp = new GraphQlFingerprinter(UrlBuilder.build(endpointUrl));
+        Object[] arguments = new Object[2];
+        GraphQlFingerprinter.setAppConsumer(
+                (site, engine) -> {
+                    arguments[0] = site.toString();
+                    arguments[1] = engine;
+                });
```

Comment:
Used an array because of the whole lambda 'must be final or effectively final' restriction.

Reply:
I guess alternatively I could have created an assertion method that accepts the two values?